packages/desktop-client/src/components/transactions/TransactionsTable.jsx

```diff
@@ -562,17 +564,46 @@ function PayeeCell({
         }
       }}
       formatter={() => getPayeePretty(transaction, payee, transferAcct)}
-      unexposedContent={props => (
-        <>
-          <PayeeIcons
-            transaction={transaction}
-            transferAccount={transferAcct}
-            onNavigateToTransferAccount={onNavigateToTransferAccount}
-            onNavigateToSchedule={onNavigateToSchedule}
-          />
-          <UnexposedCellContent {...props} />
-        </>
-      )}
+      unexposedContent={props =>
+        importedPayee ? (
+          <Tooltip
+            content={
+              <View style={{ padding: 10 }}>
+                <Text style={{ fontWeight: 'bold' }}>Imported Payee</Text>
+                <Text style={{ fontWeight: 'normal' }}>{importedPayee}</Text>
+              </View>
+            }
+            style={{ ...styles.tooltip, borderRadius: '0px 5px 5px 0px' }}
+            placement="bottom"
+            triggerProps={{ delay: 750 }}
+          >
+            <div
+              style={{ display: 'flex', alignItems: 'center', width: '100%' }}
+            >
+              <PayeeIcons
+                transaction={transaction}
+                transferAccount={transferAcct}
+                onNavigateToTransferAccount={onNavigateToTransferAccount}
+                onNavigateToSchedule={onNavigateToSchedule}
+              />
+              <UnexposedCellContent
+                {...props}
+                style={{ borderBottom: `1px dashed ${theme.pageTextSubdued}` }}
+              />
+            </div>
```

Comment:
Is it possible to pull this up into it's own variable, to avoid repeating it below?  The only difference is the `style` on the `UnexposedCellContent`, which you could easily set with a ternary.

Then the code would look more like `importedPayee ? <tooltip>$var</tooltip> : $var`.

Reply:
I'm not ignoring this, it _can't_ be hard but I appear to be fighting with React. Will give it another go with a fresh head tomorrow. I think it's something props related.

Reply:
Ha! Got it! Updated now.